compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -185,9 +184,7 @@ void RecordMinMax::saveModel(const std::string &output_model_path)
 {
   // Export to output Circle file
   luci::CircleExporter exporter;
-  CircleExpContract contract(_module.get(), output_model_path);
-
-  if (!exporter.invoke(&contract))
+  if (!exporter.exportToFile(_module.get(), output_model_path))
```

Comment:
`CircleExpContract ` is introduced to separate and now you binded again.
This is not I hoped.

Reply:
Maybe I misundertood your comment in https://github.com/Samsung/ONE/issues/3667#issuecomment-670206599.

Can you explain a little bit more?

Reply:
I think the comment is somwhat misleading (I forgot somepart of the design)
`CircleExporter` which generates export data and storing the data was decoupled by `CircleExpContract`.
This was by design.
Misleading part is that comment says storing the data to file can be included in `CircleExporter` module which I didn't think is good thing todo.

What I think is exporting to file by some methould should be independent of `CircleExporter`.
`CircleExpContract` just calls that method.

Reply:
@seanshpark  How about making `CircleFileExpContract.h` under the luci? Then, the final code would look like below.

```
#include <luci/CircleExporter.h>
#include <luci/CircleFileExpContract.h>
...

luci::CircleExporter exporter; 
CircleFileExpContract contract(_module.get(), output_model_path);

  if (!exporter.invoke(&contract))
...
```

Reply:
> How about making CircleFileExpContract.h under the luci? 

there are several modules in luci. which one or a new one?

Reply:
> there are several modules in luci. which one or a new one?


Maybe under the `luci/export/include/luci`? (where `CircleExporter.h` resides)

Reply:
OK :) let's try